test/integration/Highlight.e2e.test.js

```diff
@@ -52,4 +52,41 @@ describe('Highlights', () => {
         // Assert highlight creation mode is not active
         cy.getByTestId('bp-AnnotationsControls-highlightBtn').should('not.have.class', 'is-active');
     });
+
+    it('should restrict highlight annotation to single page', () => {
+        // Show the preview
+        cy.showPreview(Cypress.env('FILE_ID_DOC'));
+
+        // Wait for the empty highlight layer to be present
+        cy.getByTestId('ba-Layer--highlight');
+
+        let firstPageEl;
```

Comment:
The Cypress docs warn against this pattern. Can we either res-elect or alias the first page element, instead?